deployments/chart/values.yaml

```diff
@@ -4,6 +4,21 @@ image:
   repository: ghcr.io/caas-team/gokubedownscaler
   tag: "" # Dynamically set from CI
 
+arguments:
+#  - --interval=60
+
+includedResources:
+  - deployments
+  - statefulsets
+  - daemonsets
+  - rollouts
+  - horizontalpodautoscalers
+  - jobs
+  - cronjobs
+  - scaledobjects
+  - stacks
+  - poddisruptionbudgets
```

Comment:
The default is just deployments, defaults should be in the values.yaml as comments.

``` suggestion
includedResources:
  - deployments
#  - statefulsets
#  - daemonsets
#  - rollouts
#  - horizontalpodautoscalers
#  - jobs
#  - cronjobs
#  - scaledobjects
#  - stacks
#  - poddisruptionbudgets
```

OR

``` suggestion
includedResources:
  - deployments
```

Reply:
* actually don't just use the suggestion, it doesn't seem to have realised i was selecting multiple lines